src/components/main-form.js

```diff
@@ -3,11 +3,18 @@
 // https://www.nerdwallet.com/mortgages/mortgage-rates/30-year-fixed
 // https://michaelbluejay.com/house/15vs30.html
 
+// TODO
+// 1. JSLinter
+// 2. Add in content/text
+// 3. Styling Results
```

Comment:
is this what you wanted in terms of adding a TODO?

Reply:
Ah, I meant adding TODOs to all the placeholder texts, so we know to write them in the future

Reply:
oh, everything right right now is a placeholder text (even the things that sound fine - I just copied those from other websites so we should probs change them a bit). Can we just keep it at the top for now? Putting a TODO on every instance of text would be a lot of TODOs haha